src/js/components/Layer/Layer.js

```diff
@@ -1,30 +1,19 @@
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import { render, unmountComponentAtNode } from 'react-dom';
+import { createPortal } from 'react-dom';
 
 import LayerContainer from './LayerContainer';
 
 import doc from './doc';
 
-import { createContextProvider } from '../hocs';
 import { getNewContainer } from '../utils';
 
-import { deepMerge } from '../../utils';
-
 class Layer extends Component {
-  static contextTypes = {
-    grommet: PropTypes.object,
-    theme: PropTypes.object,
-  }
   static defaultProps = {
     align: 'center',
   }
 
-  componentDidMount() {
-    this.originalFocusedElement = document.activeElement;
-    this.layerContainer = getNewContainer();
-    this.renderLayer();
-  }
+  originalFocusedElement = document.activeElement;
```

Comment:
Can we access the 'document' object outside of mounting? Does this cause problems with tests or server side rendering?